CMakeLists.txt

```diff
@@ -329,3 +364,19 @@ install(DIRECTORY ${GENDIR}/uhdm/
         DESTINATION ${CMAKE_INSTALL_PREFIX}/include/uhdm/)
 install(FILES ${GENDIR}/src/UHDM.capnp
         DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/uhdm)
+
+# Generate cmake config files for reuse by downstream packages
+include(CMakePackageConfigHelpers)
+
+# generate the config file that is includes the exports
+configure_package_config_file(
```

Comment:
Maybe not in this PR, but in a follow-up: Can we also generate a [pkg-config](https://www.freedesktop.org/wiki/Software/pkg-config/) compatible [file](https://manpages.ubuntu.com/manpages/jammy/man5/pc.5.html) ? It is generally a format used in projects that do not use cmake.

 For instance [yosys-f4pga-plugins](https://github.com/chipsalliance/yosys-f4pga-plugins/blob/main/systemverilog-plugin/Makefile) uses plain makefiles, determining the headers and defines with `CXXFLAGS+=$(pkg-config --cflags uhdm)` and `LDFLAGS+=$(pkg-config --libs uhdm)` there would probably be very nice and improve packageability.

Reply:
looking into this now